lib/single-new-eks-opensource-observability-pattern/index.ts

```diff
@@ -48,31 +48,36 @@ export default class SingleNewEksOpenSourceobservabilityConstruct {
                 version: 'v5.0.0-rc3'
             }),
             new blueprints.addons.FluxCDAddOn({
-                bootstrapRepo: {
-                    repoUrl: 'https://github.com/aws-observability/aws-observability-accelerator',
-                    name: "grafana-dashboards",
-                    targetRevision: "main",
-                    path: "./artifacts/grafana-operator-manifests/eks/infrastructure"
-                },
-                fluxTargetNamespace: "grafana-operator",
-                bootstrapValues: {
-                    "AMG_AWS_REGION": region,
-                    "AMP_ENDPOINT_URL": ampPrometheusEndpoint,
-                    "AMG_ENDPOINT_URL": amgEndpointUrl,
-                    "GRAFANA_CLUSTER_DASH_URL" : clusterDashUrl,
-                    "GRAFANA_KUBELET_DASH_URL" : kubeletDashUrl,
-                    "GRAFANA_NSWRKLDS_DASH_URL" : namespaceWorkloadsDashUrl,
-                    "GRAFANA_NODEEXP_DASH_URL" : nodeExporterDashUrl,
-                    "GRAFANA_NODES_DASH_URL" : nodesDashUrl,
-                    "GRAFANA_WORKLOADS_DASH_URL" : workloadsDashUrl
-                },
+                repositories: [
+                    {
+                        name: "grafana-dashboards",
+                        namespace: undefined,
+                        repository: {
+                            repoUrl: 'https://github.com/aws-observability/aws-observability-accelerator',
+                            targetRevision: "main",
+                        },
+                        values: {
+                               "AMG_AWS_REGION": region,
+                                "AMP_ENDPOINT_URL": ampPrometheusEndpoint,
+                                "AMG_ENDPOINT_URL": amgEndpointUrl,
+                                "GRAFANA_CLUSTER_DASH_URL" : clusterDashUrl,
+                                "GRAFANA_KUBELET_DASH_URL" : kubeletDashUrl,
+                                "GRAFANA_NSWRKLDS_DASH_URL" : namespaceWorkloadsDashUrl,
+                                "GRAFANA_NODEEXP_DASH_URL" : nodeExporterDashUrl,
+                                "GRAFANA_NODES_DASH_URL" : nodesDashUrl,
+                                "GRAFANA_WORKLOADS_DASH_URL" : workloadsDashUrl
+                        },
+                        kustomizations: [{kustomizationPath:"./artifacts/grafana-operator-manifests/eks/infrastructure"}]
```

Comment:
You are missing kustomizationTargetNamespace you should have             kustomizations: [{kustomizationPath:"./artifacts/grafana-operator-manifests/eks/infrastructure",kustomizationTargetNamespace: "grafana-operator"}]